docs/getting_started.md

```diff
@@ -152,15 +151,14 @@ you can set up the pipeline necessary for deploying your ML model
 to production. The pipeline has a sequence of stages for:
 
 1. **Model Code Continuous Integration:** triggered on code change to master branch on GitHub,
```

Comment:
Perhaps add the names of the yml for each stage?

Reply:
Didn't get it. There is one yml file.